src/endpoints/workers/create_route.rs

```diff
@@ -0,0 +1,29 @@
+use super::WorkersRouteIdOnly;
+
+use crate::framework::endpoint::{Endpoint, Method};
+
+/// Create a Route
+/// Creates a route mapping the given pattern to the given script
+/// https://api.cloudflare.com/#worker-routes-create-route
+pub struct CreateRoute<'a> {
+    pub zone_identifier: &'a str,
+    pub params: CreateRouteParams,
+}
+
+impl<'a> Endpoint<WorkersRouteIdOnly, (), CreateRouteParams> for CreateRoute<'a> {
+    fn method(&self) -> Method {
+        Method::Post
+    }
+    fn path(&self) -> String {
+        format!("zones/{}/workers/routes", self.zone_identifier)
+    }
+    fn body(&self) -> Option<CreateRouteParams> {
+        Some(self.params.clone())
+    }
+}
+
+#[derive(Serialize, Clone, Debug)]
+pub struct CreateRouteParams {
```

Comment:
Could you please copy the docs for these params from the API documentation (https://api.cloudflare.com/#worker-routes-create-route)? They'd be good docstrings.

Reply:
can do